src/components/Button/Button.css

```diff
@@ -0,0 +1,62 @@
+.button {
+    margin: 5px;
+    padding: 5px;
+    border-radius: 5px;
+    height: 20px;
```

Comment:
don't need to hardcode this if you add padding. it will automatically size based on your font size